### PR TITLE
Add symlinks to no-asm libraries

### PR DESCRIPTION
**Pull request by bernedom**
Added missing symlinks to no-asm libraries so CMake can find them correctly. Without the symlinks findOpenSSL only finds the static libraries

### CODE REVIEW

**Comment by bernedom** [comment]
@bog-dan-ro @Issam-b could you have a look at this some time soon?

File: build_ssl.sh [hunk 1]
```diff
@@ -99,6 +99,7 @@ build_ssl_1_1() {
     cp libcrypto.a libssl.a "../$version_out_dir/$qt_arch" || exit 1
     ln -s "../$version_out_dir/$qt_arch/libcrypto_1_1.so" "../$version_out_dir/$qt_arch/libcrypto.so"
     ln -s "../$version_out_dir/$qt_arch/libssl_1_1.so" "../$version_out_dir/$qt_arch/libssl.so"
+    ln -s "../$version_out_dir/include" "../$version_out_dir/$qt_arch/include"
```
Comment (Issam-b) [inline build_ssl.sh:102]: why include needs to be added to each abi dir?

Comment (bernedom) [reply build_ssl.sh:102]: So it works out if the boxwith the findopenssl module of CMake. 

Like this the `OPENSSL_ROOT_DIR` can be set to the ABI folder and CMake will automatically resolve the include folder when using `find_package(OpenSSL)`

Comment (Issam-b) [reply build_ssl.sh:102]: Okay, that makes sense, thanks!

**Pull request merged**